Vagrantfile

```diff
@@ -66,11 +66,11 @@ Vagrant.configure(2) do |config|
   #   push.app = "YOUR_ATLAS_USERNAME/YOUR_APPLICATION_NAME"
   # end
 
-  # if Vagrant.has_plugin?("vagrant-proxyconf")
-  #   config.proxy.http = "http://username:password@host:port/"
-  #   config.proxy.https = "http://username:password@host:port/"
-  #   config.proxy.no_proxy = "localhost,127.0.0.1"
-  # end
+  if Vagrant.has_plugin?("vagrant-proxyconf")
+    config.proxy.http = "http://fj1738bt:0019970807@kmt.proxy.nic.fujitsu.com:8080"
+    config.proxy.https = "http://fj1738bt:0019970807@kmt.proxy.nic.fujitsu.com:8080"
+    config.proxy.no_proxy = "personium.example.com,127.0.0.1"
+  end
 
```

Comment:
Please fix only "localhost".
Please check the difference before the pull request.

Reply:
Fixed it.